pkg/resources/management.cattle.io/v3/fleetworkspace/validator.go

```diff
@@ -1,10 +1,11 @@
 package fleetworkspace
```

Comment:
Can you rename this file and the test file to `mutator.go` and `mutator_test.go`? I think this was a mistake by someone else when the file was first created.

Reply:
done in https://github.com/rancher/webhook/pull/260/commits/bc3715088473c65ec63c87a6da489240e56dbaef